### PR TITLE
[ASM] - Cortex ASM pack updated with Azure content

### PR DESCRIPTION
**Pull request by BigEasyJ**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
[EXPANDR-1436](https://jira-hq.paloaltonetworks.local/browse/EXPANDR-1436)
[EXPANDR-1600](https://jira-hq.paloaltonetworks.local/browse/EXPANDR-1600)

## Description

- Update `Cortex ASM - ASM Alert` playbook to have Azure as an option in the automated remediation conditional.
- Update `Cortex ASM - Enrichment` playbook to have  Azure option to conditional and `Cortex ASM - Azure Enrichment` playbook.
- Update `Cortex ASM - Remediation` playbook to have Azure option to conditional and Azure Network Security Group Remediation playbook.
- Add new Cortex ASM - Azure Enrichment playbook.

## Screenshots

Cortex_ASM_-_Azure_Enrichment_Updated_Feb_02_2023
Cortex_ASM_-_Enrichment_Updated_Feb_02_2023
Cortex_ASM_-_Remediation_Updated_Feb_02_2023

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests (performed internally)
- [x] Documentation

### CODE REVIEW

**Comment by BigEasyJ** [comment]
CC: @johnnywilkes @capanw

**Comment by BigEasyJ** [comment]
CC @johnnywilkes @capanw for review

**Comment by johnnywilkes** [comment]
@BigEasyJ , please make changes in Jira when possible: https://qadium.atlassian.net/wiki/spaces/PM/pages/2383872027/XSOAR+XSIAM+Dev+Standards#Jira-Fields-to-Keep-Track-of-ASM-PRs

**Comment by johnnywilkes** [comment]
@BigEasyJ , as we talked about in standup, can you also change the following to true as well (Same as "Skip this branch if this automation/playbook is unavailable" option in UI)"
- https://github.com/demisto/content/blob/54df7db0915efd2710fb728fdd0d90c61b577cb5/Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml#L65
- https://github.com/demisto/content/blob/54df7db0915efd2710fb728fdd0d90c61b577cb5/Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation.yml#L118

**Review by johnnywilkes — COMMENTED** [review]
this is my review

File: Packs/CortexAttackSurfaceManagement/README.md [hunk 1]
```diff
@@ -23,82 +26,112 @@ Automated remediation is only possible when the right conditions are met.  These
...
 
 
 #### GenerateASMReport
+
 This automation helps generate an ASM alert summary report with important information found via the playbook run.
 ![GenerateASMReport](https://raw.githubusercontent.com/demisto/content/d6d88d2066ef1f0868e8e61c5f20a71766f3cae1/Packs/CortexAttackSurfaceManagement/doc_files/GenerateASMReport.png)
 
 #### SnmpDetection
+
 This automation checka if SNMP is enabled or not and gets the running version on the remote server.
```
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/README.md]: resolved

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation.yml [hunk 2]
```diff
@@ -269,14 +283,98 @@ tasks:
...
+      {
+        "position": {
+          "x": 1100,
+          "y": 450
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
```
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation.yml]: resolved

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml [hunk 3]
```diff
@@ -0,0 +1,934 @@
...
+      {
+        "position": {
+          "x": 450,
+          "y": -190
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
```
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml]: resolved

**Comment by johnnywilkes** [comment]
> @BigEasyJ , as we talked about in standup, can you also change the following to true as well (Same as "Skip this branch if this automation/playbook is unavailable" option in UI)"
> 
> * https://github.com/demisto/content/blob/54df7db0915efd2710fb728fdd0d90c61b577cb5/Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml#L65
> * https://github.com/demisto/content/blob/54df7db0915efd2710fb728fdd0d90c61b577cb5/Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation.yml#L118

resolved

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_5_0.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+- Added the Cortex ASM - Azure Enrichment playbook as a remediation option.
+
+##### Cortex ASM - Remediation
+
+- Added the Azure Network Security Group Remediation playbook as a remediation option.
+
+##### Cortex ASM - GCP Enrichment
+
+- Update the GCP - Enrichment playbook task to skip this branch if this automation/playbook is unavailable
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_5_0.md:17]: Change to:
- Updated the GCP - Enrichment playbook task to skip this branch if this automation/playbook is unavailable.

This should resolve https://github.com/demisto/content/actions/runs/4080101970/jobs/7032208152

File: Packs/CortexAttackSurfaceManagement/pack_metadata.json [hunk 5]
```diff
@@ -55,8 +55,9 @@
         "ServiceNow",
         "Nmap",
         "Tenable_io",
+        "Azure-Enrichment-Remediation",
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/pack_metadata.json:58]: at line 45 add:
"Azure-Enrichment-Remediation": {
        "mandatory": false,
        "display_name": "Azure Enrichment and Remediation"
    }

File: Packs/CortexAttackSurfaceManagement/README.md [hunk 6]
```diff
@@ -23,82 +26,112 @@ Automated remediation is only possible when the right conditions are met.  These
...
+#### Cortex ASM - Remediation
+
 Playbook that is used as a container folder for all remediation of ASM alerts.	
-![Cortex ASM - Remediation](https://raw.githubusercontent.com/demisto/content/23747a450237bb3762d7ec7788d5ff582c8576db/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Remediation.png)	
+![Cortex ASM - Remediation](https://raw.githubusercontent.com/demisto/content/23747a450237bb3762d7ec7788d5ff582c8576db/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Remediation.png)
+
+#### Cortex ASM - Azure Enrichment
+
+Playbook that given the IP address enriches Azure information relevant to ASM alerts.	
+![Cortex ASM - Azure Enrichment](https://raw.githubusercontent.com/demisto/content/23747a450237bb3762d7ec7788d5ff582c8576db/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Azure_Enrichment.png)
```
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/README.md:107]: please change link to:
https://raw.githubusercontent.com/demisto/content/69b67c9a2330cbe9002d6d0ec8f32dbccb0c6877/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Azure_Enrichment.png

That seems to work

File: Packs/CortexAttackSurfaceManagement/README.md [hunk 7]
```diff
@@ -23,82 +26,112 @@ Automated remediation is only possible when the right conditions are met.  These
...
   - Tenable.io Assets
   - GCP IAM
+  - Azure IAM
 - Indicators of a non-production host:
   - "dev" or related words found in environment-related tags associated with the asset (case insensitive)
   - Has an active "DevelopmentEnvironment" classification from processing of public data
   
 ### Playbooks
+
+- [What does this pack do?](#what-does-this-pack-do)
```
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/README.md:45]: not resolved

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation.yml [hunk 8]
```diff
@@ -36,13 +36,13 @@ tasks:
...
-    taskid: 6ae52c8e-9574-4dcd-8167-da8bc501096f
+    taskid: 11104cdb-0212-4229-814f-c0d68fcbc21c
     type: playbook
     task:
-      id: 6ae52c8e-9574-4dcd-8167-da8bc501096f
+      id: 11104cdb-0212-4229-814f-c0d68fcbc21c
       version: -1
       name: GCP - Firewall Remediation
-      description: Replace current firewall rules with limited access firewall rules.
+      description: This playbook adds new firewall rules with access only from  private ip address range and blocks traffic that's exposed to public internet. For example, if RDP is exposed to the entire world, this playbook adds new firewall rules that only allows traffic from private ip address and blocks rest of the RDP traffic.
```
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation.yml]: John, we actually need to revert this. Apologies. It seems like the description was updated on the child pack and not sure why it never made it to master:

[content/Packs/GCP-Enrichment-Remediation/Playbooks/GCP_-_Firewall_Remediation.yml](https://github.com/demisto/content/blob/54df7db0915efd2710fb728fdd0d90c61b577cb5/Packs/GCP-Enrichment-Remediation/Playbooks/GCP_-_Firewall_Remediation.yml#L4)

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml [hunk 9]
```diff
@@ -652,7 +672,48 @@ tasks:
...
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      RemoteIP:
+        complex:
+          root: inputs.RemoteIp
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml:704]: Change this to: inputs.RemoteIP
this is how it is specified in playbook inputs and causing 2nd error in https://app.circleci.com/pipelines/github/demisto/content/231876/workflows/b391dc0e-22a8-44d0-ae08-cae7d5bd3d71/jobs/650542

**Review by johnnywilkes — CHANGES_REQUESTED** [review]
another round

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml (hunk 9 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml]: resolved in [Update input for Cortex ASM playbook](https://github.com/demisto/content/pull/24310/commits/40d107b39cd4653abe8e2fb89cb07126d9d6dcb5)

File: Packs/CortexAttackSurfaceManagement/README.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/README.md]: resolved in
https://github.com/demisto/content/pull/24310/commits/05e7231e87e8d08e66f2058411b27754fba3f52a

File: Packs/CortexAttackSurfaceManagement/README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/README.md]: resolved in 
Update TOC and Azure playbook png

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml [hunk 10]
```diff
@@ -0,0 +1,934 @@
...
+    "paper": {
+      "dimensions": {
+        "height": 1915,
+        "width": 1870,
+        "x": -300,
+        "y": -320
+      }
+    }
+  }
+inputs:
```
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml:922]: resolved in https://github.com/demisto/content/pull/24310/commits/ad0eb0dbd83d8dd1fe510dc59cdbbbf00f40f234

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_5_0.md (hunk 4 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_5_0.md]: resolved in https://github.com/demisto/content/pull/24310/commits/abb84a9697d0903de6ff9ebb9928f6e08e3f1e1b

File: Packs/CortexAttackSurfaceManagement/pack_metadata.json (hunk 5 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/pack_metadata.json:62]: resolved in https://github.com/demisto/content/pull/24310/commits/c88af851fee1e0dde19ab7d443813494a48c2f27

File: Packs/CortexAttackSurfaceManagement/README.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/README.md]: @BigEasyJ , looks better but I think you removed for the parent playbook, please re-add:
`- [Cortex ASM - ASM Alert](#cortex-asm---asm-alert)`

[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/README.md]: resolved in
[Update TOC in README. Add missing link.](https://github.com/demisto/content/pull/24310/commits/79e053838402bf4f20c93c89bb752e80ff6b2fff)

**Review by johnnywilkes — APPROVED** [review]
@BigEasyJ , great work!  This is approved.

@ilaner , can you please assign @ShirleyDenkberg  for documentation review?

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml [hunk 11]
```diff
@@ -0,0 +1,934 @@
+id: Cortex ASM - Azure Enrichment
+version: -1
+name: Cortex ASM - Azure Enrichment
+description: Given the IP address this playbook enriches Azure information relevant to ASM alerts.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml:4]:
```suggestion
description: Given the IP address, this playbook enriches Azure information relevant to ASM alerts.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml [hunk 12]
```diff
@@ -0,0 +1,934 @@
...
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: 81d10059-2900-4307-8b6c-3a65752e9cbe
+    type: regular
+    task:
+      id: 81d10059-2900-4307-8b6c-3a65752e9cbe
+      version: -1
+      name: Set service owner grid field
+      description: Sets  the potential Azure service owner to grid field for ASM service owner object.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml:219]:
```suggestion
      description: Sets the potential Azure service owner to the grid field for the ASM service owner object.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml [hunk 13]
```diff
@@ -0,0 +1,934 @@
...
+    isautoswitchedtoquietmode: false
+  "12":
+    id: "12"
+    taskid: c801e484-f37c-45c2-823f-e874b71de29a
+    type: regular
+    task:
+      id: c801e484-f37c-45c2-823f-e874b71de29a
+      version: -1
+      name: Set system IDs grid field (NSG)
+      description: Sets Azure NSG information to grid field for ASM system IDs object.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml:393]:
```suggestion
      description: Sets Azure NSG information to the grid field for the ASM system IDs object.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml [hunk 14]
```diff
@@ -0,0 +1,934 @@
...
+    isautoswitchedtoquietmode: false
+  "13":
+    id: "13"
+    taskid: 52559531-089d-4112-8de8-f4ff2d874911
+    type: regular
+    task:
+      id: 52559531-089d-4112-8de8-f4ff2d874911
+      version: -1
+      name: Set system IDs grid field (Resource Group)
+      description: Sets Azure resource group information to grid field for ASM system IDs object.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml:444]:
```suggestion
      description: Sets Azure resource group information to the grid field for the ASM system IDs object.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml [hunk 15]
```diff
@@ -0,0 +1,934 @@
...
+    isautoswitchedtoquietmode: false
+  "15":
+    id: "15"
+    taskid: b42c20cb-0fb7-45e7-8186-69503c51a2a6
+    type: regular
+    task:
+      id: b42c20cb-0fb7-45e7-8186-69503c51a2a6
+      version: -1
+      name: Set system IDs grid field (NIC)
+      description: Sets Azure NIC information to grid field for ASM system IDs object.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml:489]:
```suggestion
      description: Sets Azure NIC information to the grid field for the ASM system IDs object.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml [hunk 16]
```diff
@@ -0,0 +1,934 @@
...
+    isautoswitchedtoquietmode: false
+  "16":
+    id: "16"
+    taskid: 1bc6606c-ad86-4d2d-80c8-c9405f7330ca
+    type: regular
+    task:
+      id: 1bc6606c-ad86-4d2d-80c8-c9405f7330ca
+      version: -1
+      name: Set system IDs grid field (ID)
+      description: Sets Azure Compute instance ID to grid field for ASM system IDs object.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml:540]:
```suggestion
      description: Sets the Azure Compute instance ID to the grid field for the ASM system IDs object.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml [hunk 17]
```diff
@@ -0,0 +1,934 @@
...
+    isautoswitchedtoquietmode: false
+  "17":
+    id: "17"
+    taskid: 1836855c-6a0b-4a39-8092-fb4b83d291cb
+    type: regular
+    task:
+      id: 1836855c-6a0b-4a39-8092-fb4b83d291cb
+      version: -1
+      name: Set private IP grid field
+      description: Sets private IP to grid field for ASM private IP object.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml:585]:
```suggestion
      description: Sets the private IP to the grid field for the ASM private IP object.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml [hunk 18]
```diff
@@ -0,0 +1,934 @@
...
+    isautoswitchedtoquietmode: false
+  "20":
+    id: "20"
+    taskid: 75b690cd-03c8-41a6-887e-cfe86fb0cd10
+    type: regular
+    task:
+      id: 75b690cd-03c8-41a6-887e-cfe86fb0cd10
+      version: -1
+      name: Set cloud grid field
+      description: Sets Azure subscription, resource group, and location information to grid field for ASM cloud object.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml:690]:
```suggestion
      description: Sets Azure subscription, resource group, and location information to the grid field for the ASM cloud object.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml [hunk 19]
```diff
@@ -0,0 +1,934 @@
...
+    isautoswitchedtoquietmode: false
+  "23":
+    id: "23"
+    taskid: 1839aa7e-5f63-4b8c-86ae-92d2e617b964
+    type: regular
+    task:
+      id: 1839aa7e-5f63-4b8c-86ae-92d2e617b964
+      version: -1
+      name: Set system IDs grid field (Compute Name)
+      description: Sets Azure Compute instance name to grid field for ASM system IDs object.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml:752]:
```suggestion
      description: Sets the Azure Compute instance name to the grid field for the ASM system IDs object.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml [hunk 20]
```diff
@@ -0,0 +1,934 @@
...
+    }
+  }
+inputs:
+- key: RemoteIP
+  value:
+    complex:
+      root: inputs
+      accessor: remoteip
+  required: false
+  description: IP address of service
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml:929]:
```suggestion
  description: IP address of the service.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment_README.md [hunk 21]
```diff
@@ -0,0 +1,32 @@
+Given the IP address this playbook enriches Azure information relevant to ASM alerts.
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+* Azure - Enrichment
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment_README.md:7]:
```suggestion
Azure - Enrichment
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment_README.md [hunk 22]
```diff
@@ -0,0 +1,32 @@
+Given the IP address this playbook enriches Azure information relevant to ASM alerts.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment_README.md:1]:
```suggestion
Given the IP address, this playbook enriches Azure information relevant to ASM alerts.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment_README.md [hunk 23]
```diff
@@ -0,0 +1,32 @@
...
+
+### Commands
+This playbook does not use any commands.
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| RemoteIP | IP address of service | alert.remoteip | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment_README.md:24]:
```suggestion
| RemoteIP | IP address of the service. | alert.remoteip | Optional |
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml [hunk 24]
```diff
@@ -652,7 +672,48 @@ tasks:
...
+    isautoswitchedtoquietmode: false
+  "69":
+    id: "69"
+    taskid: 437703be-f9d0-42eb-864c-de6fd5f869e7
+    type: playbook
+    task:
+      id: 437703be-f9d0-42eb-864c-de6fd5f869e7
+      version: -1
+      name: Cortex ASM - Azure Enrichment
+      description: Given the IP address this playbook enriches Azure information relevant to ASM alerts.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml:693]:
```suggestion
      description: Given the IP address, this playbook enriches Azure information relevant to ASM alerts.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation.yml (hunk 8 above)
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation.yml:45]:
```suggestion
      description: This playbook adds new firewall rules with access only from a private IP address range and blocks traffic that's exposed to the public internet. For example, if RDP is exposed to the entire world, this playbook adds new firewall rules that only allow traffic from a private IP address and blocks the rest of the RDP traffic.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation.yml [hunk 25]
```diff
@@ -258,8 +272,92 @@ tasks:
...
+  "6":
+    id: "6"
+    taskid: f0cba2e4-b4de-450d-806a-3145e1ce4999
+    type: playbook
+    task:
+      id: f0cba2e4-b4de-450d-806a-3145e1ce4999
+      version: -1
+      name: Azure - Network Security Group Remediation
+      description: |-
+        This playbook adds new Azure Network Security Groups (NSG) rules to NSGs attached to a NIC. The new rules will give access only to a private IP address range and block traffic that's exposed to the public internet ([using the private IP of the VM as stated in Azure documentation](https://learn.microsoft.com/en-us/azure/virtual-network/network-security-groups-overview)). For example, if RDP is exposed to the public internet, this playbook adds new firewall rules that only allows traffic from private IP address and blocks the rest of the RDP traffic.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation.yml:295]:
```suggestion
        This playbook adds new Azure Network Security Groups (NSG) rules to NSGs attached to a NIC. The new rules will give access only to a private IP address range and block traffic that's exposed to the public internet ([using the private IP of the VM as stated in Azure documentation](https://learn.microsoft.com/en-us/azure/virtual-network/network-security-groups-overview)). For example, if RDP is exposed to the public internet, this playbook adds new firewall rules that only allow traffic from a private IP address and blocks the rest of the RDP traffic.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation.yml [hunk 26]
```diff
@@ -258,8 +272,92 @@ tasks:
...
+
+        Conditions and limitations:
+        - Limited to one resource group.
+        - 200 Azure rules viewed at once to find offending rule.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation.yml:299]:
```suggestion
        - 200 Azure rules viewed at once to find the offending rule.
```

File: Packs/CortexAttackSurfaceManagement/README.md [hunk 27]
```diff
@@ -23,82 +26,105 @@ Automated remediation is only possible when the right conditions are met.  These
...
-#### Cortex ASM - Remediation	
+#### Cortex ASM - Remediation
+
 Playbook that is used as a container folder for all remediation of ASM alerts.	
-![Cortex ASM - Remediation](https://raw.githubusercontent.com/demisto/content/23747a450237bb3762d7ec7788d5ff582c8576db/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Remediation.png)	
+![Cortex ASM - Remediation](https://raw.githubusercontent.com/demisto/content/23747a450237bb3762d7ec7788d5ff582c8576db/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Remediation.png)
+
+#### Cortex ASM - Azure Enrichment
+
+Playbook that given the IP address enriches Azure information relevant to ASM alerts.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/README.md:99]:
```suggestion
Given the IP address, this playbook enriches Azure information relevant to ASM alerts.	
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_5_0.md [hunk 28]
```diff
@@ -0,0 +1,21 @@
+#### Playbooks
+
+##### Cortex ASM - ASM Alert
+
+- Added an option for Azure in the automated remediation conditional.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_5_0.md:5]:
```suggestion
Added an option for Azure in the automated remediation conditional.
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_5_0.md [hunk 29]
```diff
@@ -0,0 +1,21 @@
...
+
+##### Cortex ASM - Enrichment
+
+- Added the Cortex ASM - Azure Enrichment playbook as a remediation option.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_5_0.md:9]:
```suggestion
Added the Cortex ASM - Azure Enrichment playbook as a remediation option.
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_5_0.md [hunk 30]
```diff
@@ -0,0 +1,21 @@
...
+
+##### Cortex ASM - Remediation
+
+- Added the Azure Network Security Group Remediation playbook as a remediation option.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_5_0.md:13]:
```suggestion
Added the Azure Network Security Group Remediation playbook as a remediation option.
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_5_0.md [hunk 31]
```diff
@@ -0,0 +1,21 @@
...
+
+##### Cortex ASM - GCP Enrichment
+
+- Updated the GCP - Enrichment playbook task to skip this branch if this automation/playbook is unavailable.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_5_0.md:17]:
```suggestion
Updated the GCP - Enrichment playbook task to skip this branch if this automation/playbook is unavailable.
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_5_0.md [hunk 32]
```diff
@@ -0,0 +1,21 @@
...
+
+##### New: Cortex ASM - Azure Enrichment
+
+- Created a new playbook that takes in an IP address and enriches Azure information relevant to ASM alerts. (Available from Cortex XSOAR 6.5.0).
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_5_0.md:21]:
```suggestion
Created a new playbook that takes in an IP address and enriches Azure information relevant to ASM alerts. (Available from Cortex XSOAR 6.5.0).
```

**Comment by ShirleyDenkberg** [comment]
@johnnywilkes @ilaner Doc review completed.

**Pull request merged**